### PR TITLE
Update specs

### PR DESCRIPTION
**Pull request by nisarg73**
1. Add `priority` fields.
2. Remove `useDropdown` in preview/publish.
3. Correct all draggable and required fields in specs.
4. Use `isPercentage` boolean in `PreviousEducation`'s display

### CODE REVIEW

**Comment by algomaster99** [comment]
@nisarg73 is this done?

**Comment by nisarg73** [comment]
@algomaster99 No, I'll be adding one commit on this PR only for hanfling `verified` stuff.

File: src/components/displayComponents/previousEducation.js [hunk 1]
```diff
@@ -13,13 +13,22 @@ const PreviousEducation = props => {
...
           <List.Content>
             <div>
-              {props.item.degree} - {graduationOptionsMap[props.item.graduation]} in {props.item.field}
-              <p styleName="style.gray">
-                {props.item.institute + ", "}
+              {props.item.degree} -{' '}
+              {graduationOptionsMap[props.item.graduation]} in{' '}
+              {props.item.field}
+              <div styleName="style.gray">
+                {props.item.institute + ', '}
```
Comment (algomaster99) [inline src/components/displayComponents/previousEducation.js:20]: Use template literals

File: src/components/displayComponents/previousEducation.js [hunk 2]
```diff
@@ -13,13 +13,22 @@ const PreviousEducation = props => {
...
                 {props.item.year}
                 <br />
-                CGPA {props.item.cgpa}
-              </p>
+                {props.item.isPercentage ? (
+                  (props.item.percentage &&
+                  <div styleName="style.gray">
+                    Percentage: {props.item.percentage}%
+                  </div>
+                )) : ((props.item.cgpa &&
```
Comment (algomaster99) [inline src/components/displayComponents/previousEducation.js:28]: Why 2 `(`?

File: src/constants/initial.js [hunk 3]
```diff
@@ -2,12 +2,22 @@
...
+      visibility: true
+    },
     links: []
   },
   achievement: {
     update: false,
-    data: { achievement: "", id: -1, visibility: true },
+    data: {
+      achievement: "",
+      id: -1,
```
Comment (algomaster99) [inline src/constants/initial.js:17]: What is this for?

File: src/styles.css [hunk 4]
```diff
@@ -22,6 +22,10 @@
   max-height: 80vh;
 }
 
+.formMinWidth {
```
Comment (algomaster99) [inline src/styles.css:25]: Remind me what was this for

**Review by algomaster99 — CHANGES_REQUESTED** [review]

File: src/components/displayComponents/previousEducation.js (hunk 1 above)
Comment (nisarg73) [reply src/components/displayComponents/previousEducation.js:20]: This was original implementation of SHP, should I be changing this?

File: src/components/displayComponents/previousEducation.js (hunk 2 above)
Comment (nisarg73) [reply src/components/displayComponents/previousEducation.js:28]: Sorry, I think I got confused in nested ternary operator. Will rectify this.

File: src/constants/initial.js (hunk 3 above)
Comment (nisarg73) [reply src/constants/initial.js:17]: This has initial default data for all the empty forms, eariler `priority` field was missing at multiple places, this is for those missing `priorty` fields.

File: src/styles.css (hunk 4 above)
Comment (nisarg73) [reply src/styles.css:25]: This ensures that form for all the sections have same width. Please refer https://github.com/IMGIITRoorkee/omniport-frontend-faculty-profile/issues/8 for this.

**Pull request merged**